pkg/k8splan/watcher.go

```diff
@@ -136,7 +136,7 @@ func (w *watcher) start(ctx context.Context) {
 
 	hasRunOnce := false
 
-	core.Secret().OnChange(ctx, "secret-watch", func(s string, secret *v1.Secret) (*v1.Secret, error) {
+	core.Secret().OnChange(ctx, "secret-watch", func(_ string, secret *v1.Secret) (*v1.Secret, error) {
```

Comment:
why is this being introduced in a migration PR?

Reply:
This PR also bumps the `golangci-lint` to the latest version where golint is deprecated, so replaces golint with revive, and fix some syntax errors reported by revive